x-pack/plugins/security_solution/public/common/components/ml/tables/convert_anomalies_to_users.test.ts

```diff
@@ -0,0 +1,265 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { mockAnomalies } from '../mock';
+import { cloneDeep } from 'lodash/fp';
+import { convertAnomaliesToUsers, getUserNameFromEntity } from './convert_anomalies_to_users';
+import { AnomaliesByUser } from '../types';
+
+describe('convert_anomalies_to_users', () => {
+  let anomalies = cloneDeep(mockAnomalies);
+
+  beforeEach(() => {
+    anomalies = cloneDeep(mockAnomalies);
+  });
+
+  test('it returns expected anomalies from a user', () => {
+    const entities = convertAnomaliesToUsers(anomalies);
+    const expected: AnomaliesByUser[] = [
+      {
+        anomaly: {
```

Comment:
Can you please store the values that do not change in a const and spread it in each test so we can easily see which properties are changing/being tested?